### ISSUE TITLE
Error handling for Stress Test

### ISSUE THREAD

**Issue by markfields** [microsoft/FluidFramework#5304]
Hook into warning/error events on Container, ensure we have unhandled promise rejections on, etc.

**Comment by markfields** [comment]
Bonus: Also handle errors better in e2e tests

**Comment by markfields** [comment]
My plan here is to throw together some simple fault injection mechanism in a branch and run the pipeline against that branch to generate various interesting errors, and see which ones appear in telemetry v. failing the test run, and open issues based on those repro cases.

**Comment by markfields** [comment]
There are some obvious cases to cover too:
* unhandled exceptions/rejections via Node's unhandledExceptionMonitor event
* Container warning event
* Container closing with error

There are a few other parts of the system that emit error/warning events so I may look over them too.

**Comment by markfields** [comment]
Moving to April, while also breaking out into smaller items, some of which may happen in March. This will hopefully just close before April actually happens.

**Comment by markfields** [comment]
Closing this.  I opened issues for logging uncaught exceptions/rejections, but I don't think there are other proactive steps to take.

We are getting more and more real data from the tests running - especially as the stress tests get more interesting with fault injection and leave/rejoin - and as we work through those error logs we'll find cases to fix up in terms of how the test behaves relative to what was logged.
